src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+
+    private Node<K, V>[] table;
+    private int size;
+    private int capacity;
+    private final float loadFactor;
+    private int threshold;
+
+    private static class Node<K, V> {
+        final K key;
+        V value;
+        Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    public MyHashMap() {
+        this.capacity = DEFAULT_CAPACITY;
+        this.loadFactor = DEFAULT_LOAD_FACTOR;
+        this.table = new Node[capacity];
+        this.threshold = (int) (capacity * loadFactor);
+    }
 
     @Override
-    public void put(K key, V value) {
+    public int getSize() {
+        return size;
+    }
+
+    private int hash(K key) {
+        return key == null ? 0 : Math.abs(key.hashCode() % capacity);
+    }
+
+    private void resize() {
+        capacity = capacity * 2;
+        Node<K, V>[] newTable = new Node[capacity];
```

Comment:
Private methods and classes should be after public ones in your class. Move the `resize` method below the `getValue` method.